aesara/scalar/basic.py

```diff
@@ -2878,7 +2875,11 @@ def c_code(self, node, name, inputs, outputs, sub):
         return f"{z} = 1.0 / {x};"
 
 
-inv = Inv(upgrade_to_float, name="inv")
+reciprocal = Reciprocal(upgrade_to_float, name="reciprocal")
+
+# These are deprecated and will be removed
+Inv = Reciprocal
+inv = reciprocal
```

Comment:
Maybe add the [`deprecated`](https://github.com/pymc-devs/aesara/blob/master/aesara/utils.py#L149) decorator from `aesara.util`?

Reply:
The `deprecated` wrapper works fine for deprecating _functions_, but this is a class and a class instance, and we don't want to unnecessarily break things by changing the expected types of these objects.

We would need to use something like `__init_subclass__` or a meta class solution to add deprecation warnings in these cases.